code/DebugBarDatabaseCollector.php

```diff
@@ -34,12 +34,13 @@ public function collect()
         // Check for excessive number of queries
         $dbQueryWarningLevel = DebugBar::config()->warn_query_limit;
         if ($dbQueryWarningLevel && $data['nb_statements'] > $dbQueryWarningLevel) {
+            $helpLink = DebugBar::config()->performance_guide_link;
```

Comment:
Thanks for the feedback. This approach isn't bad practice at all, is valid use of the SS API and is consistent with the rest of the existing module in terms of how config is used.